ci/build_wheel_libcuspatial.sh

```diff
@@ -3,4 +3,31 @@
 
 set -euo pipefail
 
+git clone \
+  --branch multiple-file-keys \
+  https://github.com/jameslamb/dependency-file-generator.git \
+  /tmp/rapids-dependency-file-generator
+
+pip uninstall --yes rapids-dependency-file-generator
+pip install /tmp/rapids-dependency-file-generator
+
```

Comment:
```suggestion
```

TODO: remove this before merging

Reply:
removed this now that https://github.com/rapidsai/dependency-file-generator/pull/115 has been merged, `rapids-dependency-file-generator` 1.16.0 has been released, and all new CI images have been built (https://github.com/rapidsai/ci-imgs/actions/runs/11461526811)